.drone/scripts/publish.py

```diff
@@ -16,12 +16,13 @@
     pkgver = json["current_pkgver"]
     pkgrel = json["current_pkgrel"]
 
-if commit_branch == "stable":
-    package_name = "makedeb"
-else:
-    package_name = f"makedeb-{commit_branch}"
+# if commit_branch == "stable":
+#     package_name = "makedeb"
+# else:
+#     package_name = f"makedeb-{commit_branch}"
```

Comment:
We still need these lines to publish different releases of makedeb to the ProGet instance.